src/txn/txn_ckpt.c

```diff
@@ -845,6 +846,7 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
         WT_ERR(__wt_txn_checkpoint_log(session, full, WT_TXN_LOG_CKPT_START, NULL));
 
     __checkpoint_timing_stress(session);
+    time_start_hs = __wt_clock(session);
```

Comment:
I don't think this is the correct location for this, this would be timing the checkpoint for all files not including the metadata. I think we only want to time the checkpoint for the actual history file.

Reply:
Oh thanks, didn't notice that, I placed the start and stop in the conditional statement now.